### PR TITLE
Feature: Review Modal

### PR DESCRIPTION
**Pull request by dterr**
### MVP for Review Modal
This adds a body of the Modal that switches visibility with the results body. Triggering the reviews will switch off the results body with the reviews body, which remains uncleared (like a cache) but invisible. There will be a back arrow that will clear the review body but make the result visible again. 

``` index-script.js ``` : Add functions for review modal to turn on
``` modal-overlay.css ``` : Add stylings for the new review modal body and container elements.
``` index.html ``` : Add containers for the review modals within body.

### CODE REVIEW

File: src/main/webapp/index-script.js [hunk 1]
```diff
@@ -314,3 +314,66 @@ function generateResult(place) {
...
+ * Puts the review text in a <p> element
+ */
+function generateReview(review) {
+  const reviewEntry = document.createElement('li');
+
+  const reviewGrid = document.createElement('div');
+  reviewGrid.className += review-grid;
+
+  const reviewText = document.createElement('p');
+  reviewText.innerHTML += "review.text placeholder = Great food and even better service.  Beer pizza Italian Beef. Who could ask for more.";
```
Comment (ccjeane) [inline src/main/webapp/index-script.js:363]: Yuck! Beer pizza italian beef does not sound appetizing!

Comment (ccjeane) [reply src/main/webapp/index-script.js:363]: The power of commas...

Comment (dterr) [reply src/main/webapp/index-script.js:363]: Boilerplate review from Aurelio's Pizza in Mokena. 
_American Heartland Cuisine_
**That is vestigial testing code.**

Comment (ccjeane) [reply src/main/webapp/index-script.js:363]: ### Cool ### 
**COOL**
_COOL_

File: src/main/webapp/index-script.js [hunk 2]
```diff
@@ -314,3 +314,95 @@ function generateResult(place) {
...
 }
+
+/** Pseudomaster Review Function
+ * One central function that is called to trigger entire review interface
+ */
+function showReviews(placeID) {
+  let reviewsArray = fetchReviews(placeID);
+  displayReviewModal();
+  populateReviews(reviewsArray);
+
```
Comment (fyordan) [inline src/main/webapp/index-script.js:325]: delete empty newline

File: src/main/webapp/modal-overlay.css [hunk 3]
```diff
@@ -103,6 +107,28 @@
```
Comment (fyordan) [inline src/main/webapp/modal-overlay.css:107]: delete this line

**Pull request merged**